devcycle.rst

```diff
@@ -215,7 +215,19 @@ Repository Administration
 '''''''''''''''''''''''''
 
 The source code is currently hosted on `GitHub
-<https://github.com/python/cpython>`_.
+<https://github.com/python/cpython>`_ in the `Python organization <https://github.com/python/>`_.
+
+Organization Repository Policy
+------------------------------
+
+Within the `Python organization <https://github.com/python/>`_, repositories are expected to fall within three general categories:
+
+1. The refence implementation of Python and related repositories (i.e. `CPython <https://github.com/python/cpython>`_)
+2. Reference implementations of PEPs (e.g. `mypy <https://github.com/python/mypy>`_)
+3. Tooling and support around CPython and the language (e.g. `python.org repository <https://github.com/python/pythondotorg>`_)
+
```

Comment:
I'd like to see explicit inclusion of generalized PSF and PSF Infrastructure related projects.